.github/workflows/continuous-integration.yml

```diff
@@ -59,14 +59,20 @@ jobs:
           arguments: shellcheck
         if: runner.os == 'Linux'
       - name: Report jacoco coverage
+        id: jacoco_report
         uses: gradle/gradle-build-action@v2
         env:
           ORG_GRADLE_PROJECT_epApiVersion: ${{ matrix.epVersion }}
-          COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
         with:
-          arguments: coveralls
+          arguments: :code-coverage-report:clean codeCoverageReport --no-build-cache
         continue-on-error: true
         if: runner.os == 'Linux' && matrix.java == '11' && matrix.epVersion == '2.21.1' && github.repository == 'uber/NullAway'
+      - name: Upload coverage reports to Codecov
+        uses: codecov/codecov-action@v3
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          files: ./code-coverage-report/build/reports/jacoco/codeCoverageReport/codeCoverageReport.xml
+        if: steps.jacoco_report.conclusion == 'success'
```

Comment:
Does `steps.jacoco_report.conclusion != 'success'` if that step does not run?  Just checking, as that step only runs on a certain OS, Java version, etc. (see line 69)

Reply:
Yes, this gets skipped if the previous step doesn't run. Initially I had the same conditions as the previous step, but that felt redundant for this step. 
However, I just realized that the previous step has `continue on error` as true, so I have changed this to use `outcome` instead of `conclusion` so that it runs iff the code coverage report succeeds. For `conclusion == success`, it will run even if the step fails with an error which can cause issues uploading and break the workflow.